UniWatch/Controllers/AccountController.cs

```diff
@@ -149,14 +153,24 @@ public ActionResult Register()
         [ValidateAntiForgeryToken]
         public async Task<ActionResult> Register(RegisterViewModel model)
         {
-            if (ModelState.IsValid)
+            if(ModelState.IsValid)
             {
-                var user = new ApplicationUser { UserName = model.Email, Email = model.Email };
+                var user = new ApplicationUser { UserName = model.FirstName, Email = model.Email, PhoneNumber = model.PhoneNumber };
```

Comment:
We don't really make use of the user name and we don't require it to be unique. When logged in, it displays "hello {username}" at the top and that looked funny with the email, so I changed it.